stress-example/text/index.yml

```diff
@@ -7,26 +7,35 @@ with:
 pods:
   - name: segmenter
     show_exc_info: true
-    uses: pods/segment.yml
+    uses: segment.yml
     read_only: true
+    shards: $JINA_SHARDS_ENCODER
+    host: $JINA_ENCODER_HOST
+    port_expose: 8000
+    timeout_ready: -1
   - name: encoder
     polling: any
-    show_exc_info: true
-    uses: pods/encoder.yml
     shards: $JINA_SHARDS_ENCODER
-    timeout_ready: 600000
+    host: $JINA_ENCODER_HOST
+    port_expose: 8000
+    show_exc_info: true
+    uses: encoder.yml
+    timeout_ready: -1
     read_only: true
   - name: chunk_idx
     polling: any
     show_exc_info: true
-    uses: pods/chunk.yml
+    uses: chunk.yml
     shards: $JINA_SHARDS_INDEXERS
+    host: $JINA_VEC_INDEXER_HOST
+    port_expose: 8000
+    timeout_ready: -1
   - name: redis
     polling: any
     show_exc_info: true
-    uses: pods/redis.yml
+    uses: redis.yml
     shards: $JINA_SHARDS_INDEXERS
-    needs: gateway
```

Comment:
do not remove this, this is important for the test, the most u can do is comment it out